### PR TITLE
#355: Add new data resource

### PR DESCRIPTION
**Pull request by Daenarys8**
Ping: #355, 
@antagomir

### CODE REVIEW

File: 04_containers.Rmd [hunk 1]
```diff
@@ -347,7 +347,25 @@ library(curatedMetagenomicData)
 tse <- curatedMetagenomicData("Vatanen*", dryrun = FALSE, counts = TRUE)
 ```
 
+###  Human microbiome compendium
 
+[MicroBioMap](https://seandavi.github.io/MicroBioMap/) dataset includes 
+over 170k samples of publicly available 16S rRNA amplicon sequencing data, 
+all processed using the same pipeline and reference database[@Abdill2023].
+
+Load the compendium using:
```
Comment (antagomir) [inline 04_containers.Rmd:356]: "After installing the MicroBioMap package (see the original website for instructions), you can load the compendium with:"

File: 04_containers.Rmd [hunk 2]
```diff
@@ -347,7 +347,25 @@ library(curatedMetagenomicData)
...
+library(MicroBioMap)
+cpd <- getCompendium()
+```
+
+The resulting object is a TreeSummarizedExperiment object. Currently, 
+the "tree" part of the TreeSummarizedExperiment is not populated, but 
+that is on the roadmap[@compendiumpackage].
+
+After loading the compendium, you will have immediate access to nearly 
+170,000 microbiome samples.
```
[the remark this answers is not in the document]
Comment (antagomir) [reply 04_containers.Rmd:368]: Add also:

"For more use examples in R/Bioconductor, see the MicroBioMap [vignette](https://github.com/seandavi/MicroBioMap/blob/devel/vignettes/overview.Rmd)."

File: book.bib [hunk 3]
```diff
@@ -1457,4 +1457,24 @@ @article{Schloss2023
...
+	year = {2023},
+	doi = {10.1101/2023.10.11.560955},
+	publisher = {Cold Spring Harbor Laboratory},
+	URL = {https://www.biorxiv.org/content/early/2023/10/11/2023.10.11.560955},
+	eprint = {https://www.biorxiv.org/content/early/2023/10/11/2023.10.11.560955.full.pdf},
+	journal = {bioRxiv}
+}
+
+@Manual{compendiumpackage,
+  title = {MicroBioMap: Access the microbiome compendium from R},
```
Comment (antagomir) [inline book.bib:1475]: Put MicroBioMap in curly brackets, also "R". Then the rendering keeps the relevant uppercases.

**Review by antagomir — COMMENTED** [review]
Seems good. Some minor comments.

There are also some small changes in many files that are unrelated to this new data set, not sure why?

**Comment by Daenarys8** [comment]
I was surprised at the changes as well. but it came from streamlining with source(build.R)

**Comment by antagomir** [comment]
Confirm when changes ready.

**Comment by Daenarys8** [comment]
PTAL...

**Comment by antagomir** [comment]
@Daenarys8 what is the status of this one?

Could you resolve the conflicts and see if we can complete this. 

Or if anything unclear, let's try to troubleshoot and get this done.

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
Awesome, let's see how it looks live!

**Pull request merged**